charts/hono/templates/_helpers.tpl

```diff
@@ -252,6 +252,21 @@ resource-limits:
 {{- end }}
 {{- end }}
 
+{{/*
+Configuration for the image name for services.
+The scope passed in is expected to be a dict with keys
+- "dot": the root scope (".") and
+- "conf": the configuration of the component
+*/}}
+{{- define "hono.imageName" -}}
+{{- $tag := ( default .dot.Chart.AppVersion ( default .dot.Values.honoImagesTag .dot.Values.adapters.http.imageTag ) ) -}}
+{{- if ( contains "quarkus" .dot.Values.honoImagesType ) }}
```

Comment:
sure this is *contains*? looks like this branch is only applicable to JVM Quarkus images but not native images. or am I mistaken?

Reply:
By default with Quarkus you can build either JVM or native image and they will end up having the same name. So in that sense this will work with the current Hono implementation. We should discuss how we're gonna build and name images in Hono and adapt this according to that.

Reply:
I see